### PR TITLE
Fuck windows

### PR DESCRIPTION
**Pull request by Debonaircorsiar**
## Fuck windows

Some chuckle fuck thought it was a good idea to make reinforced windows and reinforced plasmawindows robust as shit against damage and I'm taking that away from them, mostly.

Reinforced(and plastitanium) windows had 11 damage deflection, to the smoothbrains reading this; what that means is when you hit the windows ~~it subtracts 11 damage from the attack regardless of the source~~ **eliminates melee damage lower than 11** before its armor value is taken into consideration. This means anything less then a toolbox, which does 12 damage, will do no damage to the window. In comparison; a fire extinguisher does 10 damage, a wrench and crowbar do about 5 damage each, a potted plant does about 10 damage, a chair does 8, etc; little to no actual round start object is capable of breaking a reinforced window other then the tool box and welder. Which is fairly absurd, so I lowered the value or reinforced glass down to a little less then half so that people can actually break into places if they need too, but will still take some time to get through. 

I also did the same with reinforced plasma glass, as it was effectively immune to damage with its damage deflection of 21, I lowered it to 11, meaning only the fire axe is able to break through it. By lowering it its making it possible to break but still very annoying to do so.

## Why It's Good For The Game

Theres alot of instances where breaking into or out of places is possibly the only option you have, and by making windows nearly unbreakable limits the choices you have, and by lowering the value it offers some more possibilities for traitors to do potential traitor shinanigains.

## Changelog
:cl:
Balance: Reduced reinforced, plasma, and reinforced plasma glass damage deflection.
/:cl:

Edit by @Superyodeler: Corrected damage deflection, clarifies Balance, removed Removes: Mark.

### CODE REVIEW

**Comment by triplezeta** [comment]
cool

**Comment by dee-dubya** [comment]
The reason they were made that way was because reinforced windows used to be a complete joke. They were used on secure areas that were otherwise lined with the extremely durable reinforced walls, yet they could be defeated by some minor toolboxing - or worse, three clicks(screwdriver>crowbar>screwdriver).

There was another PR with this goal in mind, and collectively we decided that nerfing reinforced glass was a good idea, but a mapping project to replace many instances of reinforced glass with plasmaglass in secure areas on maps would need to follow.

**Comment by Picklesoup** [comment]
> 
> 
> The reason they were made that way was because reinforced windows used to be a complete joke. They were used on secure areas that were otherwise lined with the extremely durable reinforced walls, yet they could be defeated by some minor toolboxing - or worse, three clicks(screwdriver>crowbar>screwdriver).
> 
> There was another PR with this goal in mind, and collectively we decided that nerfing reinforced glass was a good idea, but a mapping project to replace many instances of reinforced glass with plasmaglass in secure areas on maps would need to follow.

I can sorta agree with this. Rwindows are really irritatingly stronk, but there's also the issue of making them too much of a weak point in a structure. I think it should be a weak point and a predictable route someone could follow when they just need to get into a place quickly and don't care about stealth. But I also think there maybe should be another level of secure that uses plasmawindows like the armory.

**Comment by Debonaircorsiar** [comment]
> EDIT: Also to be clear, Rwindows should take more than just a toolbox to get rid of but should still be reasonably quick to get through with actual serious weapons. 

This PR still keeps that, it just makes it possible with less. it takes, depending on your mood, around 75-100 hits with a toolbox to break a reinforced window with a toolbox, which takes about 1 and a half minutes to 2 and a half minutes to break through. So breaking through is still gonna be difficult with the highest end in greytide technology and deconstructing it is much much faster, this just opens an alternative.

> There was another PR with this goal in mind, and collectively we decided that nerfing reinforced glass was a good idea, but a mapping project to replace many instances of reinforced glass with plasmaglass in secure areas on maps would need to follow.

plasmaglass windows are easier to break then Rwindows,  the only difference between a plasmaglass window and a regular window is 25 integrity, unless you mean R-Plasmaglass windows, which I agree should be slapped onto places like the armory. This PR would make R-Plasmaglass about as strong as R-glass is currently, meaning nothing less then a spear or fire axe will get through it quickly.

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -366,7 +366,7 @@
 	armor = list("melee" = 80, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 25, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 100)
 	max_integrity = 75
 	explosion_block = 1
-	damage_deflection = 11
+	damage_deflection = 5
```
Comment (Superyodeler) [inline code/game/objects/structures/window.dm:369]:
```suggestion
	damage_deflection = 5		// Wasp Edit - Weakens R-Windows
```

File: code/game/objects/structures/window.dm [hunk 2]
```diff
@@ -492,7 +492,7 @@
 	heat_resistance = 50000
 	armor = list("melee" = 80, "bullet" = 20, "laser" = 0, "energy" = 0, "bomb" = 60, "bio" = 100, "rad" = 100, "fire" = 99, "acid" = 100)
 	max_integrity = 500
-	damage_deflection = 21
+	damage_deflection = 11
```
Comment (Superyodeler) [inline code/game/objects/structures/window.dm:495]:
```suggestion
	damage_deflection = 11		// Wasp Edit - Weakens R-Windows
```

File: code/game/objects/structures/window.dm [hunk 3]
```diff
@@ -700,7 +700,7 @@
 	smooth = SMOOTH_TRUE
 	canSmoothWith = null
 	explosion_block = 3
-	damage_deflection = 11 //The same as normal reinforced windows.3
+	damage_deflection = 5
```
Comment (Superyodeler) [inline code/game/objects/structures/window.dm:703]:
```suggestion
	damage_deflection = 5		// Wasp Edit - Weakens R-Windows
```

**Comment by Debonaircorsiar** [comment]
> 
> 
> margular

If you say this one more time I'm commiting kneecap removal from you.

**Comment by Superyodeler** [comment]
Margular

**Comment by Debonaircorsiar** [comment]
Thats it, I'm removing you from code next.

**Pull request merged**